src/client/helpers.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphQLTransportWSClient } from './client';
+import { print } from 'graphql/language/printer';
+
+/**
+ * @deprecated This method becomes deprecated in the new package graphql-transport-ws.
+ * Start using the GraphQLTransportWSClient to make queries, mutations and subscriptions over websockets.
+ */
+
+// Quick way to add the subscribe and unsubscribe functions to the network interface
+export function addGraphQLSubscriptions(networkInterface: any, wsClient: GraphQLTransportWSClient): any {
+  console.warn('This method becomes deprecated in the new package graphql-transport-ws. Start using the ' +
```

Comment:
'This method' => 'addGraphQLSubscriptions', the user on console shouldn't guess ;)